build-logic/src/main/kotlin/dokkabuild.base.gradle.kts

```diff
@@ -21,3 +22,9 @@ tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
     isReproducibleFileOrder = true
 }
+
+val integrationTestPreparation by tasks.registering {
+    description =
+        "lifecycle task for preparing the project for integration tests (for example, publishing to the test Maven repo)"
+    group = VERIFICATION_GROUP
+}
```

Comment:
I'll be using this task more in later PRs to help manage local Maven publishing of the included-builds.